aiida/cmdline/commands/computer.py

```diff
@@ -123,6 +118,76 @@ def prompt_for_computer_configuration(computer):
                 print >> sys.stderr, "Enter '?' for help".format(e.message)
 
 
+@verdi_computer.command('enable')
+@click.pass_context
+@click.option(
+    '-u',
+    '--only-for-user',
+    type=types.UserParamType(),
+    required=False,
+    help="Enable a computer only for the given user. If not specified, enables the computer globally.")
+@arguments.COMPUTER()
+@with_dbenv()
+def enable_computer(ctx, only_for_user, computer):
+    """Enable a computer"""
+    from aiida.common.exceptions import NotExistent
+
+    if only_for_user is None:
+        if computer.is_enabled():
+            echo.echo_info("Computer '{}' already enabled.".format(computer.name))
+        else:
+            computer.set_enabled_state(True)
+            echo.echo_info("Computer '{}' enabled.".format(computer.name))
+    else:
+        try:
+            authinfo = computer.get_authinfo(only_for_user)
+        except NotExistent:
+            echo.echo_critical("User with email '{}' is not configured for computer '{}' yet.".format(
+                only_for_user.email, computer.name))
+
+        if not authinfo.enabled:
+            authinfo.enabled = True
+            echo.echo_info("Computer '{}' enabled for user {}.".format(computer.name, only_for_user.get_full_name()))
+        else:
+            echo.echo_info("Computer '{}' was already enabled for user {} {}.".format(
+                computer.name, only_for_user.first_name, only_for_user.last_name))
+
+
+@verdi_computer.command('disable')
+@click.pass_context
+@click.option(
+    '-u',
+    '--only-for-user',
+    type=types.UserParamType(),
+    required=False,
+    help="Disable a computer only for the given user. If not specified, disables the computer globally.")
+@arguments.COMPUTER()
+@with_dbenv()
+def disable_computer(ctx, only_for_user, computer):
+    """Disable a computer. Useful, for instance, when a computer is under maintenance."""
+    from aiida.common.exceptions import NotExistent
+
+    if only_for_user is None:
+        if not computer.is_enabled():
+            echo.echo_info("Computer '{}' already disabled.".format(computer.name))
+        else:
+            computer.set_enabled_state(False)
+            echo.echo_info("Computer '{}' disabled.".format(computer.name))
```

Comment:
Maybe for the cases where the (desired) change is actually effectuated, it would be good to use `echo_success` instead of `echo_info`?